### DISCUSSION TITLE
移除 babel-plugin-import

### DISCUSSION THREAD

**Discussion by WT-Road** [ant-design/ant-design#39197] [Q&A]
"umi": "^3.5.8",
"antd": "^5.0.0",

升级到V5后报错
umi dev启动后控制台报错
Module not found: Can't resolve 'antd/es/auto-complete/style'

目前项目能正常运行,点击其它页面也正常,新添加了Tour漫游式引导后,点击页面后报错
[@umijs/runtime] load component failed Error: Module "./antd/es/tour/style" does not exist in container.
while loading "./antd/es/tour/style" from webpack/container/reference/mf

查了下文档需要移除 babel-plugin-import
可是我并没有配置,全局搜索也没找到
"plugins": [
- ["import", { "libraryName": "antd", "libraryDirectory": "lib"}, "antd"],
]

**Comment by WT-Road** [comment]
Umis升级到4.x解决了

**Reply by liuxi0811** [reply]
是咋解决的

**Reply by WT-Road** [reply]
> 是咋解的

我把umi从@3升到了@4就解决了
